packages/web3-plugin-example/src/index.ts

```diff
@@ -0,0 +1,16 @@
+/*
```

Comment:
why we need this empty file?

Reply:
Originally it was because the `main` entry file in the `package.json` was pointing to this file, but I've moved the `web3-plugin-example` to the `tools/` directory, as it shouldn't ever be published to NPM and thus the `main` entry is not needed